docs/support/tutorials/dask-python.md

```diff
@@ -68,10 +78,17 @@ __master job batch job file__
 ### Load the python-data module
 module load python-data
 
-### Run the Dask example. We also give script the project name and number of worker jobs
-srun python dask_multinode.py <YOUR-PROJECT> 4
+### Run the Dask example. We also give script the project name and number of worker SLURM jobs
+srun python dask_multinode.py <YOUR-PROJECT>
 ```
 
+The worker jobs are defined inside the Python file started by master SLURM job, for further details see: [Dask Jobqueue configurations documentation](https://jobqueue.dask.org/en/latest/configuration-setup.html).
+
+* `cores` - How many cores per node to use? In bigger jobs one worker SLURM job should fill the whole HPC node, ie 40 cores in Puhti.
+* `processes` - How many Python processes per node to use?
+* `memory`- How much memory per node to use? This should be enough for all Dask workers in that node. If unsure, try with cores*6Gb.
+* `walltime` - Reserve long enough time, one worker may handle several delayed functions, if number of workers is smaller than number of delayed functions.
+
 __simple python script__
 ```
```

Comment:
Can python code be formatted as python code instead of plain formatting? Like here: https://www.markdownguide.org/extended-syntax/#syntax-highlighting  but python instead of json

Reply:
Changed to Python styling.